### ISSUE TITLE
purge unused storage

### ISSUE THREAD

**Issue by yubrew** [public-awesome/launchpad#368]
https://twitter.com/larry0x/status/1565461890897219584?s=20&t=jC-G_1uJePR-4jp4RiINxw

maybe this can be limited to creators or operators?

**Comment by jhernandezb** [comment]
yeah something I've thinking specially for whitelists, and minters.

**Comment by shanev** [comment]
Should also change whitelists to work with merkle trees instead at some point.

**Comment by yubrew** [comment]
Making separate issue for using merkletree for lists
